test/system/my/solutions_information_bar_test.rb

```diff
@@ -126,27 +126,6 @@ class My::SolutionsInformationBarTest < ApplicationSystemTestCase
     assert_selector ".notifications-bar", text: "Your exercise has been submitted successfully."
   end
 
-  test "In independent mode with slots" do
```

Comment:
Why are these two tests redundant now?

Reply:
I've tested these edge cases in the helper test already. I've also added a system test to request mentor feedback via the information bar.